### DISCUSSION TITLE
Improve > improve?

### DISCUSSION THREAD

**Discussion by sedixonUW** [kogalur/randomForestSRC#418] [General]
Hello! I'm trying to use your multivariate random forest tool and tune my parameters. The model is running okay with an OOB R^2 of .27. When I try to run the tune() function I get the following error: 

> o <- tune(Multivar(proportionBA_RA + proportionBA_DF, proportionBA_WH, proportionBA_BM,proportionBA_RC)
+           ~ Age_2021 + tpa_calc + elev_2020 + slope_2020 + total_BA + site_index + num_species, data = cfi_plots_forest, sampsize = 189, doBest=TRUE)

Error in if (Improve > improve) { : missing value where TRUE/FALSE needed

I've checked my data and I don't have any NA values. Is there another reason why I could be getting this error and do you have any suggestions? Thanks so much!

**Comment by ishwaran** [comment]
The multivariate formula only accepts variables that are defined in the data frame.  You seem to be trying to define a variable by summing two variables which is not allowed; for example

`proportionBA_RA + proportionBA_DF`

The solution is to add the two variables together and define a new variable in the data frame, say

`cfi_plots_forest$newvariable <- proportionBA_RA + proportionBA_DF`
